go-controller/pkg/util/go_ovn.go

```diff
@@ -122,3 +122,15 @@ func initGoOvnUnixClient(address, db string) (goovn.Client, error) {
 	klog.Infof("Created OVNDB UNIX client for db: %s", db)
 	return ovndbclient, nil
 }
+
```

Comment:
nit, a comment explaining at high level what the function does?

Reply:
done. thanks